### PR TITLE
use redis cache as celery broker in local environments

### PR DESCRIPTION
**Pull request by nickpell**
Required for celery 4.  Does not affect any production environments.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1485,6 +1485,7 @@ def _determine_couch_databases(couch_databases):
 Thanks,
 The CommCare HQ Team"""
 
+BROKER_URL = BROKER_URL or redis_cache['LOCATION']
```
[the remark this answers is not in the document]
Comment (emord) [reply settings.py]: `redis_cache` exits in localsettings but not settings, so I don't believe that this is guaranteed to work. Any reason not to just change the orginal line to `BROKER_URL = redis://localhost:6379/0` which can then be updated by localsettings

[the remark this answers is not in the document]
Comment (nickpell) [reply settings.py]: ```'redis://localhost:6379/0'``` doesn't work for my setup, so I was hoping to provide something that would work at least some of the time.  I'm fine with your approach too.

[the remark this answers is not in the document]
Comment (nickpell) [reply settings.py]: I'm inclined to stick with the current version, to avoid getting too deep in making settings changes.

**Review by emord — APPROVED** [review]

**Comment by nickpell** [comment]
@dimagi/team-commcare-hq Docker for Mac users may need to add ```BROKER_URL = 'redis://<docker-machine-ip>:6379/0'``` to localsettings.

**Pull request merged**